plugins/Wallet/js/reducers/sendprompt.js

```diff
@@ -1,16 +1,19 @@
 import { Map } from 'immutable'
-import { SET_SEND_AMOUNT, SET_SEND_ADDRESS } from '../constants/wallet.js'
+import { SET_SEND_AMOUNT, SET_SEND_ADDRESS, START_SEND_PROMPT } from '../constants/wallet.js'
 
 const initialState = Map({
 	sendaddress: '',
 	sendamount: '',
+	currencytype: 'siacoins',
 })
 export default function sendPromptReducer(state = initialState, action) {
 	switch (action.type) {
 	case SET_SEND_AMOUNT:
 		return state.set('sendamount', action.amount)
 	case SET_SEND_ADDRESS:
 		return state.set('sendaddress', action.address)
+	case START_SEND_PROMPT:
+		return state.set('sendtype', action.currencytype)
```

Comment:
shouldn't this be `currencyType`?
As a whole, I'm not sure I understand why the currencyType is part of the prompt's state, since it's not modifiable (right?). Shouldn't it be passed as a prop instead?